dev/diagrams/generate_airflow_architecture_diagram.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from pathlib import Path
+
+from diagrams import Cluster, Diagram, Edge
+from diagrams.custom import Custom
+from diagrams.onprem.client import User
+from diagrams.onprem.database import PostgreSQL
+from diagrams.programming.flowchart import MultipleDocuments
+from rich.console import Console
+
+console = Console(width=400, color_system="standard")
+
+IMAGE_NAME = "basic_airflow_architecture_diagram.png"
+
+LOCAL_DIR = Path(__file__).parent
+AIRFLOW_SOURCES_ROOT = Path(__file__).parent.parent.parent
+DOCS_IMAGES_DIR = AIRFLOW_SOURCES_ROOT / "docs" / "apache-airflow" / "img"
+
+PYTHON_MULTI_PROCESS_LOGO = AIRFLOW_SOURCES_ROOT / "dev" / "images" / "python_multiprocess_logo.png"
+
+
+def generate_basic_airflow_diagram():
+    with Diagram(name="Basic Airflow Architecture Diagram", show=False, direction="TB", curvestyle="curved"):
+        operations_user = User("Operations User")
+        with Cluster("UI"):
+            with Cluster("Webserver"):
+                webservers = Custom("Webservers", PYTHON_MULTI_PROCESS_LOGO.as_posix())
```

Comment:
```suggestion
                webservers = Custom("Webserver(s)", PYTHON_MULTI_PROCESS_LOGO.as_posix())
```

Would write `(s)` to indicate it can be one or more